csds-client/client/v3/client_test.go

```diff
@@ -35,6 +35,15 @@ func TestParseNodeMatcherWithFile(t *testing.T) {
 	if !clientUtil.ShouldEqualJSON(t, string(get), want) {
 		t.Errorf("NodeMatcher = \n%v\n, want: \n%v\n", string(get), want)
 	}
+	wantNode := "{\"id\":\"fake_client_node_id\"}"
+	getNode, errNode := protojson.Marshal(&c.node)
+	if err != nil {
+		t.Errorf("Parse NodeMatcher Error: %v", errNode)
```

Comment:
nit: consider different error message here and below (compared to the error messages above), to easily identify where an error is originating.

Reply:
Updated the error message.